build/package/helm/gateway/files/policies/policies.rego

```diff
@@ -0,0 +1,79 @@
+package m8.authz
+
+import future.keywords.in
+
+default authorized = false
+
+# list of allowed paths for any user
+allowed_paths := [
+	"/gateway.Gateway/",
+	"/gateway.ClusterAuth/GetAuthToken/",
+	"/domain.CommandHandlerExtensions/",
+	"/domain.Cluster/GetByName",
+]
+
+scoped_paths := [{"path": "/scim/", "scope": "WRITE_SCIM"}]
+
+command_path := "/eventsourcing.CommandHandler/Execute"
+
+scope_system = "system"
+
+scope_tenant = "tenant"
+
+role_admin = "admin"
+
+# check if system admin
+is_system_admin {
+	some role in input.User.Roles
+	role.Scope == scope_system
+	role.Name == role_admin
+	print(input.User.Name, "is system admin")
+}
+
+# check if user is tenant admin and adjusts rolebindings of other users of the tenant
```

Comment:
Can you please explain what you mean by "adjusts rolebindings ..."? As far as I understood we are only checking here, right?

Reply:
The comment means that this checks if the user is tenant admin and the user is trying to "change" rolebindings of other users for the tenant he or she is admin for.